packages/@ourworldindata/grapher/src/controls/ChartIcons.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
+import {
+    faChartLine,
+    faChartColumn,
+    faEarthAmericas,
+} from "@fortawesome/free-solid-svg-icons"
+import { ChartTypeName } from "../core/GrapherConstants"
+
+export const chartIcons: Record<ChartTypeName, JSX.Element> = {
+    // line chart
+    [ChartTypeName.LineChart]: <FontAwesomeIcon icon={faChartLine} />,
+
+    // bar charts
+    [ChartTypeName.DiscreteBar]: <FontAwesomeIcon icon={faChartColumn} />,
+    [ChartTypeName.StackedBar]: <FontAwesomeIcon icon={faChartColumn} />,
+    [ChartTypeName.StackedDiscreteBar]: (
+        <FontAwesomeIcon icon={faChartColumn} />
+    ),
+
+    // world map
+    [ChartTypeName.WorldMap]: <FontAwesomeIcon icon={faEarthAmericas} />,
```

Comment:
Note for the reviewers: I think `ChartTypeName.WorldMap` is legacy code? For completeness sake, I assigned an icon...